public/locales/fr/explore.json

```diff
@@ -1,5 +1,6 @@
 {
   "appName": "Explorateur IPLD",
+  "tabName": "Explorer",
```

Comment:
This is just for demo purposes, so preview of French language in this PR fully works even without `tx pull -a`